### PR TITLE
Dls 4424 Play 28 Upgrade 

### PR DESCRIPTION
**Pull request by gavinharris43**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -13,45 +13,37 @@ val appName = "help-to-save-stride-frontend"
...
   "com.typesafe.play" %% "play-test" % PlayVersion.current % test,
   "com.typesafe.play" %% "play-ws" % PlayVersion.current % test
+
 )
+//
+//  "org.scalatestplus"      %% "scalatestplus-mockito"       % "1.0.0-M2"          % "test",
+//  "com.typesafe.play"      %% "play"                        % "2.8.8"             % "test",
+//  "com.miguno.akka"        %% "akka-mock-scheduler"         % "0.5.5"             % "test",
+//  "uk.gov.hmrc"            %% "bootstrap-test-play-28"      % "5.16.0"            % "test",
+//
```
Comment (rambabu-posa) [inline build.sbt:46]: Please remove these comments

File: build.sbt [hunk 2]
```diff
@@ -13,45 +13,37 @@ val appName = "help-to-save-stride-frontend"
...
-  "uk.gov.hmrc" %% "domain" % "5.11.0-play-26",
+  "uk.gov.hmrc" %% "govuk-template" % "5.72.0-play-28",
+  "uk.gov.hmrc" %% "mongo-caching" % "7.0.0-play-28",
+  "uk.gov.hmrc" %% "play-ui" % "9.7.0-play-28",
+  "uk.gov.hmrc" %% "bootstrap-frontend-play-28" % "5.2.0",
+  "uk.gov.hmrc" %% "domain" % "6.2.0-play-28",
   "org.typelevel" %% "cats-core" % "2.6.0",
   "com.github.kxbmap" %% "configs" % "0.6.1",
   compilerPlugin("com.github.ghik" % "silencer-plugin" % "1.7.1" cross CrossVersion.full),
   "com.github.ghik" % "silencer-lib" % "1.7.1" % Provided cross CrossVersion.full
```
Comment (rambabu-posa) [inline build.sbt:26]: It's a minor change, not a mandatory.
If possible change silencer version to 1.7.5 for both entries.

**Review by rambabu-posa — COMMENTED** [review]
LGTM with few minor changes

File: build.sbt (hunk 1 above)
Comment (gavinharris43) [reply build.sbt]: done

File: build.sbt (hunk 2 above)
Comment (gavinharris43) [reply build.sbt]: done

**Review by rambabu-posa — APPROVED** [review]
LGTM

**Pull request merged**